test/BinderSpec.js

```diff
@@ -357,7 +357,7 @@ describe('Binder', function() {
     });
   });
 
-  it('ShoulIgnoreVbNonBindable', inject(function($rootScope, $compile) {
+  it('ShouldIgnoreVbNonBindable', inject(function($rootScope, $compile) {
```

Comment:
This is a minor change to the testsuite's categorization system. It's unlikely that it would break anything, although historical data about failures would be disconnected.